### PR TITLE
Moved all the service commands under experimental build tag

### PR DESCRIPTION
**Pull request by mavenugo**
1. In order to support the docker experimental feature build, moving the
    service commands under experimental tag. Please refer to :
    https://github.com/docker/docker/pull/13338/ for more information

2. Also includes the Formatting fix from #203. Thanks to @nerdalert 

Signed-off-by: Madhu Venugopal <madhu@docker.com>

### CODE REVIEW

**Comment by mavenugo** [comment]
ping @mrjana @aboch

File: api/api.go [hunk 1]
```diff
@@ -107,7 +107,7 @@ func (h *httpHandler) initRouter() {
 	h.r = mux.NewRouter()
 	for method, routes := range m {
 		for _, route := range routes {
-			r := h.r.Path(route.url).Methods(method).HandlerFunc(makeHandler(h.c, route.fct))
```
Comment (aboch) [inline api/api.go]: Looks like this change comes from PR #205

File: api/api_test.go [hunk 2]
```diff
@@ -1136,7 +1136,7 @@ func TestHttpHandlerUninit(t *testing.T) {
 	}
 
 	rsp := newWriter()
```
Comment (aboch) [inline api/api_test.go]: Same for this file, it belongs to change  in PR #205

File: cmd/dnet/dnet.go [hunk 3]
```diff
@@ -113,7 +113,7 @@ func (d *dnetConnection) dnetDaemon() error {
 	}
 	httpHandler := api.NewHTTPHandler(controller)
 	r := mux.NewRouter().StrictSlash(false)
-	post := r.PathPrefix("/networks").Subrouter()
+	post := r.PathPrefix("/{.*}/networks").Subrouter()
```
Comment (aboch) [inline cmd/dnet/dnet.go]: Looks like this change comes from PR #205

**Comment by mavenugo** [comment]
@aboch fixed it. PTAL.

**Comment by aboch** [comment]
LGTM

**Comment by mrjana** [comment]
LGTM

**Pull request merged**